packages/main/src/MultiComboBox.js

```diff
@@ -735,13 +739,13 @@ class MultiComboBox extends UI5Element {
 
 	rootFocusIn() {
```

Comment:
The handlers rootFocusIn/Out are not semantically correct anymore, should become inputFocusIn/Out. 
The "focused' property is fine, as the Input's CSS is applied when the attribute is in place -  :host([focused]).

What is the reason for the order in the test spec. Is it for readability, or the test is failing for some reason?

Reply:
The test was failing for some reason which I thought at first is a test issue, since the behaviour it tests was working. However I later noticed that I have broken the behaviour when Backspace is pressed while the Tokenizer does not have enough space to be expanded. That is the reason why the test was passing when I moved it, since the window sizes were not yet changed. Anyway, now should be fine.